src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private Random random = new Random();
+    private Ball ball = new Ball();
```

Comment:
make this variable local

Reply:
In this case, we won't to create each time its instance when call this method?